snyk/models.py

```diff
@@ -36,11 +36,16 @@ class Vulnerability(DataClassJSONMixin):
     cvssScore: Optional[str] = None
 
 
+@dataclass
+class LicenseIssue(DataClassJSONMixin):
```

Comment:
This doesn't appear to represent the full license information? I think it's worth either all or nothing here, ie. keep the todo and the raw data for the moment, or complete the rest of the fields.

```json
"licenses": {
          "type": "array",
          "items": {
            "anyOf": [
              {
                "type": "object",
                "properties": {
                  "id": {
                    "type": "string"
                  },
                  "url": {
                    "type": "string"
                  },
                  "title": {
                    "type": "string"
                  },
                  "from": {
                    "type": "array"
                  },
                  "package": {
                    "type": "string"
                  },
                  "version": {
                    "type": "string"
                  },
                  "severity": {
                    "type": "string"
                  },
                  "language": {
                    "type": "string"
                  },
                  "packageManager": {
                    "type": "string"
                  },
                  "isIgnored": {
                    "type": "boolean"
                  },
                  "isPatched": {
                    "type": "boolean"
                  },
                  "ignored": {
                    "type": "array"
                  },
                  "patched": {
                    "type": "array"
                  }
                },
                "required": [
                  "id",
                  "url",
                  "title",
                  "from",
                  "package",
                  "version",
                  "severity",
                  "isIgnored",
                  "isPatched"
                ]
              }
            ]
          }
        }
      },
```

Reply:
Done. Additional work is needed in #34 to map the `from` attribute.
I've also submitted a support ticket to have a look at `isIgnored` and `isPatched`.